src/lib/components/buttons/flat_white.tsx

```diff
@@ -77,6 +77,7 @@ const styles = StyleSheet.create({
     alignItems: "center",
     borderColor: colors["gray-regular"],
     borderWidth: 1,
+    flex: 1,
```

Comment:
Yeah - I went back and checked after, turned out it looked bad - it's what made me decide it was time to start https://github.com/orta/jest-snapshots-svg